recipes/transdecoder/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: transdecoder
+  version: "2.1.0"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: transdecoder-2.1.0.tar.gz
+  url: https://github.com/TransDecoder/TransDecoder/archive/v2.1.0.tar.gz
+  md5: f86c688e13ced3cc51fed3d508a2ea05
+
+requirements:
+  build:
+    #  
+  run:
+    - libgcc >=4.9.1 # GOMP_4.0 requirement
```

Comment:
This is ok I think. libgcc is backward compatible, so other package depending on libgcc should still work, even if this leads to a newer libgcc being pulled.